rules/falco_rules.yaml

```diff
@@ -3054,6 +3054,16 @@
   priority: WARNING
   tags: [container, cis, mitre_lateral_movement]
 
+- rule: Unprivileged Delegation of Page Faults Handling to a Userspace Process
+  desc: Detect a successful unprivileged userfaultfd syscall which might act as an attack primitive to exploit other bugs
+  condition: >
+    evt.type = userfaultfd and
```

Comment:
Is this syscall `userfaultfd` available to consume now? I didn't find it in the syscall table.

Reply:
Hey @Kaizhe, as written in the PR corpus, such a syscall would be available once https://github.com/falcosecurity/libs/pull/50 gets merged in :)

Reply:
To test this PR is sufficient changing `cmake/modules/falcosecurity-libs.cmake` to point to the branch of that PR (also changing the SHA256 hash or de-commenting it just for testing purposes).

Reply:
Cool! I missed that, can we have a custom macro for the rule say `user_known_user_faultfd_activities` with condition `never_true`?

Reply:
Indeed we can.

It seems to me it's a good suggestion!